.gitignore

```diff
@@ -129,4 +129,5 @@ dmypy.json
 .pyre/
 .idea/
 .vscode/
-/data
\ No newline at end of file
+/data
+/projects
```

Comment:
Remove this and `projects` and `data` from .gitignore

Reply:
Removed for projects but data contains lots of files. Like PG related files..